WL4EditorWindow.ui

```diff
@@ -352,7 +353,7 @@
   </action>
   <action name="actionNew_Door">
    <property name="text">
-    <string>New Door</string>
+    <string>Door</string>
```

Comment:
Since this adds a new door, it should not change (keep it "New Door")
And the same for New Room

Reply:
Done.